### PR TITLE
Staking collection Guide

### PR DESCRIPTION
**Pull request by joshuahannan**
Closes: https://github.com/onflow/flow-core-contracts/issues/214

## Description

Adds a user guide for staking collection, explaining:
* High-level overview of staking collection
* Guide for all the transactions
* Guide for all the scripts
* List of events
* Short explanation of all the staking options

For contributor use:

- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/content/core-contracts/staking-collection.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+|**`SCO.19`**| Get Locked Tokens Used                 | [stakingCollection/scripts/get_locked_tokens_used.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/stakingCollection/scripts/get_locked_tokens_used.cdc) |
+|**`SCO.20`**| Get Unlocked Tokens Used               | [stakingCollection/scripts/get_unlocked_tokens_used.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/stakingCollection/scripts/get_unlocked_tokens_used.cdc) |
+|**`SCO.21`**| Get Machine Accounts                   | [stakingCollection/scripts/get_machine_accounts.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/stakingCollection/scripts/get_machine_accounts.cdc) |
+
+
+## Setup Transaction
+
+To setup the Staking Collection for an account, use the `SC.01` transaction.
+
+The setup process finds any node or delegator records already stored in the main account's storage, as well as any in the associated locked account. It connects them with the new Staking Collection, allowing them
```
Comment (JeffreyDoyle) [inline docs/content/core-contracts/staking-collection.mdx:73]:
```suggestion
The setup process finds any node or delegator records already stored in the main account's storage, as well as any in the associated locked account if an associated locked account exists. It connects these node and delegator records with the new Staking Collection, allowing them
```

File: docs/content/staking/staking-options.mdx [hunk 2]
```diff
@@ -15,14 +15,40 @@ including locked FLOW.
...
+The basic staking guide is detailed [here](/staking/unlocked-staking-guide/)
 
 # Locked Tokens Staking
 
-TODO
+This method is what almost all of the participants in the Flow Community sale originally used for staking.
+It involves creating a secondary account that holds the user's locked tokens from the sale.
+A capability is created for the secondary account that allows the user's main account to 
+register to stake and perform staking actions with the locked tokens 
+without allowing them to actually withdraw any of the locked tokens.
```
Comment (JeffreyDoyle) [inline docs/content/staking/staking-options.mdx:33]:
```suggestion
without allowing them to actually withdraw any of the locked tokens to their primary account.
```

File: docs/content/core-contracts/staking-collection.mdx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+The `FlowStakingCollection` contract is a contract that manages a resource containing a user's stake and delegation objects.
+
+The `FlowStakingCollection` allows a user to manage multiple active nodes or delegators 
+and interact with node or delegator objects stored in either their optional locked account
+or in the StakingCollection itself (stored in the main account).
+If a user has locked tokens, StakingCollection allows a user to interact with their locked tokens
+to perform staking actions for any of their nodes or delegators.
+
+The staking collection also manages creating a node's machine accounts if they have any collector or consensus nodes.
+It also allows them to deposit and withdraw tokens from any of their machine accounts through the staking collection.
```
Comment (JeffreyDoyle) [inline docs/content/core-contracts/staking-collection.mdx:17]: Does Staking Collection facilitate depositing tokens to Machine Accounts?

**Review by JeffreyDoyle — APPROVED** [review]
✅ Approving with just the three comments

File: docs/content/core-contracts/staking-collection.mdx (hunk 3 above)
Comment (joshuahannan) [reply docs/content/core-contracts/staking-collection.mdx:17]: No, I was assuming that we would just use a normal token transfer for that

**Pull request merged**